pkg/storage/storage.go

```diff
@@ -229,10 +251,15 @@ func (p *storage) storePodMetrics(pods []PodMetricsPoint) {
 	pointsStored.WithLabelValues("container").Set(float64(containerCount))
 }
 
-func cpuUsageOverTime(metricPoint, prevMetricPoint MetricsPoint) resource.Quantity {
+func cpuUsageOverTime(metricPoint, prevMetricPoint MetricsPoint) (*resource.Quantity, error) {
 	window := metricPoint.Timestamp.Sub(prevMetricPoint.Timestamp).Seconds()
 	cpuUsageScaled := metricPoint.CpuUsage.ScaledValue(-9)
 	prevCPUUsageScaled := prevMetricPoint.CpuUsage.ScaledValue(-9)
+
+	if cpuUsageScaled-prevCPUUsageScaled < 0 {
+		return nil, fmt.Errorf("Got negative CPU usage")
```

Comment:
This doesn't explain what happens
```suggestion
		return nil, fmt.Errorf("Unexpected decrease in cumulative CPU usage value")
```